marker_to_ee:
  rotation:
  - 0.000359962934834937
  - 0.0005630976282134442
  - -0.001974614706015697
  - 0.9999978271198944
  translation:
  - -0.021827765285817147
  - 0.0004519932661509081
  - 0.03461763979467197
camera_to_robot:
  rotation:
  - -0.5003579412296671
  - -0.4987536716475639
  - 0.5011053977218389
  - 0.49978003765708845
  translation:
  - 0.7858591230909718
  - -0.031012561989642418
  - 0.31308824851448874
